runtime/app.go

```diff
@@ -81,6 +81,12 @@ func (a *App) Load(loadLatest bool) error {
 	if len(a.config.InitGenesis) != 0 {
 		a.ModuleManager.SetOrderInitGenesis(a.config.InitGenesis...)
 		a.SetInitChainer(a.InitChainer)
+		// set export order to init order by default
+		a.ModuleManager.SetOrderExportGenesis(a.config.InitGenesis...)
+	}
+
+	if len(a.config.ExportGenesis) != 0 {
+		a.ModuleManager.SetOrderExportGenesis(a.config.ExportGenesis...)
```

Comment:
It's maybe minor, but it'd feel more correct to call `SetOrderExportGenesis` only once depending on what is set